### ISSUE TITLE
Path of War psionic archetypes not working

### ISSUE THREAD

**Issue by Pavgran** [PCGen/pcgen#2422]
Path of war archetypes for psionic classes (Pathwalker and War Soul) are basically not working.
You cannot choose known disciplines, there is no stances or maneuvers in spell list, you cannot choose primary warrior path for pathwalker.

EDIT:
Seems like discipline choice can be fixed.
In the file `pcgen/data/pathfinder/dreamscarred_press/path_of_war/pow_abilities_psi.lst`:
If you move `BONUS:ABILITYPOOL|Pathwalker Discipline|2` from `Pathwalker ~ Disciplines` ability to `Maneuvers` ability, for example, the choice of disciplines appears.
But I still don't understand why `Primary Martial Warrior Path` pool is empty and why is spell list empty after choosing disciplines.

**Comment by LegacyKing** [comment]
I fixed the first portion... Working on the other.

**Comment by Pavgran** [comment]
I've made a pull request with discipline choice fix.
Now you can choose the discipline and choose primal path for that discipline.

Spell list for maneuvers and stances still doesn't show up

**Comment by LegacyKing** [comment]
I'm trying to recall if there is a JIRA open for this...

**Comment by Pavgran** [comment]
The `ADD:SPELLCASTER|Pathwalker ADD:SPELLCASTER|Pathwalker Stances` thing is not working.
Looks like when it fires, there are no discipline spell lists available and therefore no spellcaster is added.
I've tried to make an additional ability that contains the string above and that is activated only when you have chosen at least one discipline. But that doesn't add spellcaster neither.
Then I've tried to add the above string to the martial paths (when you choose one, it is guaranteed that you already have at least one discipline chosen), and now spellcaster is added. But it doesn't appear in spell list until you add another class (any would suffice, even non-spellcaster).

In summary:
I've managed to make it work, but:
Spellcasters are added after you make the first martial path choice, AND
Spells don't appear until you add and then (optionally) remove a level from an another (or same) class.

**Comment by LegacyKing** [comment]
merged the PR

**Comment by oddmage** [comment]
I came across this issue because it's similar to what I am running into.  I have the tested on PCGen (6.08.00RC9 & 6.09.05) but I am having an issue with my Soulknife with the War Soul archetype but I cannot pick any maneuvers under spells.  I've tried adding and removing other Path of War classes, but still not changing.  Tagging @LegacyKing since I believe you are still one of the main maintainers and I wanted to thank you for that, even if you can't help with this.
